dev/ci/ci-serapi.sh

```diff
@@ -11,5 +11,7 @@ if [ "$DOWNLOAD_ONLY" ]; then exit 0; fi
 
 ( cd "${CI_BUILD_DIR}/serapi"
   make
-  dune install -p coq-serapi --prefix="$CI_INSTALL_DIR"
+  make test
+  # Not needed by any other CI job for now
+  # dune install -p coq-serapi --prefix="$CI_INSTALL_DIR"
```

Comment:
even if it's not needed we should still install I think

Reply:
Ok